CHANGELOG.md

```diff
@@ -14,6 +14,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+## [0.8.0]
+### Changed
+- *BREAKING:* Adds `optionality` to `StructField` in `partiql-types`
+- *BREAKING:* Removed `NULL` and `MISSING` types from `partiql_types::PartiQLType`
+- *BREAKING:* Removed `partiql_ast_passes::partiql_type`
```

Comment:
Looks like you need to remove these from `Unreleased` above?

Reply:
done